requirements.txt

```diff
@@ -7,3 +7,4 @@ pandas
 scikit-learn>=0.22
 scipy>=1.3.3
 threadpoolctl
+bokeh
```

Comment:
I believe that as of the latest commit, we're pulling dependencies from `tedana/info.py`

Reply:
My bad, I wasn't up to date with master. The latest commit should address this. Thank you for spotting that!